symengine/number.h

```diff
@@ -147,6 +147,7 @@ tribool is_nonpositive(const Basic &b);
 tribool is_negative(const Basic &b);
 tribool is_nonnegative(const Basic &b);
 tribool is_real(const Basic &b, const Assumptions *assumptions = nullptr);
+tribool is_complex(const Basic &b, const Assumptions *assumptions = nullptr);
```

Comment:
Where is this method implemented? I didn't see it in this PR, but maybe I missed it. I was just curious.

Reply:
It is in `test_visitors.cpp` together with the other "test"-functions related to numbers.

Reply:
Ah, I see it now. Thanks!